service/service.py

```diff
@@ -0,0 +1,120 @@
+# Copyright 2016, 2019 John J. Rofrano. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Recommendation Service
+
+Paths:
+-----
+list: HTTP GET /recommendations
+read: HTTP GET /recommendations/{rec-id}
+query: HTTP GET /recommendations?product-id=123&type=upscale&customer-id=111
+create: HTTP POST /recommendations
+update: HTTP PUT /recommendations/{rec-id}
+delete: HTTP DELETE /recommendations/{rec-id}
+success: HTTP PUT /recommendations/{rec-id}/success
+"""
+
+import os
+import sys
+import logging
+from flask import Flask, jsonify, request, url_for, make_response, abort
+from flask_api import status    # HTTP Status Codes
+from werkzeug.exceptions import NotFound
+
+# For this example we'll use SQLAlchemy, a popular ORM that supports a
+# variety of backends including SQLite, MySQL, and PostgreSQL
+from flask_sqlalchemy import SQLAlchemy
+from service.models import Pet, DataValidationError
+
+# Import Flask application
+from . import app
+
+######################################################################
+# Error Handlers
+######################################################################
```

Comment:
no need to write error handlers now, if you keep it that way, the test coverage would be poor